mlflow/tracking/_model_registry/utils.py

```diff
@@ -47,6 +47,28 @@ def set_registry_uri(uri):
                   Databricks CLI
                   `profile <https://github.com/databricks/databricks-cli#installation>`_,
                   "databricks://<profileName>".
+
+    .. code-block:: python
+        :caption: Example
+
+        import mflow
+
+        # Set model registry uri, fetch the set uri, and compare
+        # it with the tracking uri. They should be different
+        mlflow.set_registry_uri("sqlite:///tmp/registry.db")
```

Comment:
```suggestion
        mlflow.set_registry_uri("sqlite:////tmp/registry.db")
```

`"sqlite:///tmp/registry.db"` is a relative path.

Reply:
Yes, an extra `/` makes it absolute